### PR TITLE
added swagger

### PR DESCRIPTION
**Pull request by andrey11m**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -49,6 +50,21 @@
...
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-swagger2</artifactId>
+            <version>${io.springfox.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-swagger-ui</artifactId>
+            <version>${io.springfox.version}</version>
+        </dependency>
+        <dependency>
```
Comment (Estrin-MBicycle) [inline pom.xml:63]: I'm not sure you need this starter

File: src/main/java/internship/mbicycle/storify/configurations/SwaggerConfig.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package internship.mbicycle.storify.configurations;
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/configurations/SwaggerConfig.java:1]: Please rename pack from 'configurations' to 'configuration'

**Review by Estrin-MBicycle — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (andrey11m) [reply pom.xml:63]: It didn't work for me without boot starter and i found that from 2.2 version spring boot need this depency

File: src/main/java/internship/mbicycle/storify/configurations/SwaggerConfig.java (hunk 2 above)
Comment (andrey11m) [reply src/main/java/internship/mbicycle/storify/configurations/SwaggerConfig.java:1]: I took names of packs from timetracker, the name of controllers or configufations must be like controller and configufation?

Comment (OlgaEvdokimova) [reply src/main/java/internship/mbicycle/storify/configurations/SwaggerConfig.java:1]: Yes, we name packeges in singular

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -1,4 +1,7 @@
 spring:
+  mvc:
+    pathmatch:
+      matching-strategy: ant_path_matcher
```
Comment (OlgaEvdokimova) [inline src/main/resources/application.yml:4]: Are u going to use different content in your requests?

File: pom.xml (hunk 1 above)
Comment (OlgaEvdokimova) [reply pom.xml:63]: Use 3.0 version

Comment (OlgaEvdokimova) [reply pom.xml:63]: Try to use only this one dependency https://mvnrepository.com/artifact/io.springfox/springfox-boot-starter

File: src/main/resources/application.yml (hunk 3 above)
Comment (andrey11m) [reply src/main/resources/application.yml:4]: Just without it didn't work swagger for me

**Pull request closed without merging**